server/src/test/java/org/opensearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -787,4 +795,47 @@ public TransportResponse.Empty read(StreamInput in) {
         }
 
     }
+
+    public void testFollowerCheckTimeoutValue() {
```

Comment:
nit : `testFollowerCheckTimeoutValue` => `testFollowerCheckTimeoutValueDynamicUpdate`

Reply:
Since the word update already shows dynamism, `testFollowerCheckTimeoutValueUpdate` is better suited IMO.